tests/test_caldav.py

```diff
@@ -1457,10 +1457,11 @@ def testTodoDatesearch(self):
         # t6 has dtstart and due set prior to the search window, but is yearly recurring.
         # What will a date search yield?
         noexpand = self.check_compatibility_flag("no_expand")
-        todos1 = c.date_search(
```

Comment:
This code does not need to be touched.  If you look right below, the new `search`-method is already exercised in this test code.

Reply:
I believe this applies to all code in the test directory.  I went through there myself to make sure the `search`-method would do the same as `date_search`.

Reply:
so in the tests directory, I should not make any changes.

Reply:
Correct.  The date_search should be removed eventually, but only after the 1.0-release.  Still need the tests in place in case of retention issues.